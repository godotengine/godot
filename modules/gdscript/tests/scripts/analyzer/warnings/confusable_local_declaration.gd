func test():
	if true:
		var a = 1
		print(a)
	var a = 2
	print(a)
