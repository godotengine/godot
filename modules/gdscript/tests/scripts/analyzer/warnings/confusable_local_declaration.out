GDTEST_OK
~~ WARNING at line 3: (CONFUSABLE_LOCAL_DECLARATION) The variable "a" is declared below in the parent block.
1
2
